doofinder-for-woocommerce/readme.txt

```diff
@@ -83,7 +83,10 @@ Just send your questions to <mailto:support@doofinder.com> and we will try to an
 == Changelog ==
 
 = 2.0.21 =
```

Comment:
I think this should be changed to 2.0.22

Reply:
ups, you are right!